New instance parameter 'additional_groups'

ruriky opened the pull request:
User can define additional groups for instance and fenced users.
Multiple groups can be defined as an array.

This change was needed because our db2 users need be able to become instance and fenced users using sudo command. There is an easy way of allowing users to become other users which are in the same group.

crayfishx commented:
Thanks @ruriky  - seems a good feature to have - I find the naming convension of "additional groups" a bit confusing though - it suggests there is somewhere else to set groups too, which there isn't - does this make more sense to just call the parameter `groups`??

ruriky commented:
Thanks for the comment @crayfishx - that makes sense. I changed the parameter name to 'groups'. Also in README.md file I switched 'additional groups' to 'supplementary groups' as that is how the man pages seem to describe them.
